src/cdomain/value/cdomains/floatDomain.ml

```diff
@@ -618,8 +618,71 @@ module FloatIntervalImpl(Float_t : CFloatType) = struct
     | (l, _) when l > Float_t.zero -> false_zero_IInt ()
     | _ -> unknown_IInt () (**any interval containing zero has to fall in this case, because we do not distinguish between 0. and -0. *)
 
-  (**This Constant overapproximates pi to use as bounds for the return values of trigonometric functions *)
+  (**These constants over-/underpproximate pi *)
   let overapprox_pi = 3.1416
+  let underapprox_pi = 3.1415
```

Comment:
Is this still needed after adding the stub?

Reply:
Thanks for the quick changes! I was wondering if the same doesn't hold for `overapprox_pi`? Can we just use the value from the stub and round up?